### ISSUE TITLE
Wraps lines when out of room

### ISSUE THREAD

**Issue by nionio6915** [jneilliii/OctoPrint-FloatingNavbar#3]
If there is too much information to the navbar, the lines will wrap and new height of the Navbar will obscure the tabs at the top. 

In my case, loaded plugins in the Navbar are PSUControl, Octolapse, Estop, Navbar Temp and TouchUI, 
screenshot here- https://imgur.com/a/w4X2W

Toggling Full Screen navbar with alter the width of the navbar information also.

**Comment by jneilliii** [comment]
That's more of an issue with octoprint itself and not my plugin.

**Comment by foosel** [comment]
@jneilliii I disagree. That the navbar wraps, yes, that is an OctoPrint issue. That the wrapped navbar *when floated due to your plugin being installed* then makes parts of the interface inaccessible, that's an issue with your plugin.

tldr:

![image](https://user-images.githubusercontent.com/83657/37974396-220caf56-31dd-11e8-8d8f-01f30eea01f4.png)

**Comment by sbts** [comment]
As for octoprint wrapping.
I think thats a fair behaviour if there is more information to be displayed than fits on a single line.

There is only one other rational way to handle that. Continuously scroll the content but then depending on the content that may prevent access to elements when required. 

To keep this in perspective not everyone is viewing on wide screens with high resolution. 
So wrapping could happen even on a fairly standard screen

**Comment by jneilliii** [comment]
Sorry guys, missed the fact that this was on the floating navbar plugin, thought it was a comment to one of my power plug plugins. I'll adjust the offset based on navbar height insteadof being hard coded.

**Comment by jneilliii** [comment]
This issue should be fixed with latest version 0.3.0.

**Comment by sbts** [comment]
@jneilliii thanks for that.
